src/main/java/CLIOutput.java

```diff
@@ -37,43 +36,63 @@ public static void outputBoard (Board board, boolean symbol) {
         }
     }
     
-    private static getPieceSymbol (String pieceLetter, int[] pieceCoord) {
+    private static String getPieceSymbol (Piece piece) {
         // Returns the symbol for the piece
-        
-        //TODO: Figure out of piece if white or black!!!
 
-        switch(pieceLetter){
-            case "P":
-                return "♙";
-            case "R":
-                return "♖";
-            case "N":
-                return "♘";
-            case "B":
-                return "♗";
-            case "Q":
-                return "♕";
-            case "K":
-                return "♔";
-            case "p":
-                return "♟";
-            case "r":
-                return "♜";
-            case "n":          
-                return "♞";
-            case "b":
-                return "♝";
-            case "q":
-                return "♛";
-            case "k":
-                return "♚";
-            default:
-                if (pieceCoord[0]+pieceCoord[1] % 2 == 0) {
-                    return "■";
-                } else {
-                    return "□";
-                }
+        if (piece.getColor() == 'w') {
+            switch(piece.getType()){
+                case "p":
+                    return "♟";
+                case "r":
+                    return "♜";
+                case "n":          
+                    return "♞";
+                case "b":
+                    return "♝";
+                case "q":
+                    return "♛";
+                case "k":
+                    return "♚";
+            }
+        } else {
+            switch(piece.getType()){
+                case "p":
+                    return "♙";
+                case "r":
+                    return "♖";
+                case "n":
+                    return "♘";
+                case "b":
+                    return "♗";
+                case "q":
+                    return "♕";
+                case "k":
+                    return "♔";
+            }
         }
+        return " ";
+    }
+    private static String getPieceLetter (Piece piece) {
+        // Returns the symbol for the piece
 
+        if (piece.getColor() == 'w') {
+            switch(piece.getType()){
+                case "p":
+                    return "P";
+                case "r":
+                    return "R";
+                case "n":          
+                    return "N";
+                case "b":
+                    return "B";
+                case "q":
+                    return "Q";
+                case "k":
+                    return "K";
+            }
+        } else {
+            return piece.getType();
+        }
+        return " ";
     }
 }
```

Comment:
Not strictly necessary but it is standard to end files with a newline. Consider configuring IntelliJ to enforce this automatically.